Cargo.toml

```diff
@@ -14,14 +14,25 @@ readme = "README.md"
 repository = "https://github.com/static-files-rs/static-files"
 
 [features]
-default = [ "change-detection" ]
+default = ["change-detection"]
+change-detection = ["dep:change-detection"]
 
 [dependencies]
 change-detection = { version = "1.2", optional = true }
 mime_guess = "2.0"
-path-slash = "0.1"
+path-slash = "0.x"
```

Comment:
0.x? Is it a typo, or just some cargo feature? :)

Reply:
This one is tricky. It [compiles](https://github.com/nyurik/static-files/pull/1/checks) (your CI requires approval, so I ran it on my fork too), and it is [recommended](https://github.com/rhysd/path-slash?tab=readme-ov-file#installation) by the crate author, but it is not documented anywhere. The semver parser does understand it as a wildcard. I filed a bug https://github.com/dtolnay/semver/issues/322

I will use `0.2` to avoid questions